Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+COPY sphereon-pe-models-*.tgz ./
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+EXPOSE 3000
+COPY src ./src
+RUN cd src
+RUN npm start ./index.ts
```

Comment:
The start script uses nodemon which is development mode. We need to transpile it first and then run it from the build folder.